src/pages/api/chapters/[chapterId].ts

```diff
@@ -68,6 +74,62 @@ async function handler(
       } catch (e) {
         return serverErrorHandler(e, res);
       }
+    case 'GET':
+      try {
+        // Check if the provided chapter user exists
+        const prisma = new PrismaClient();
+
+        const existingChapter = await prisma.chapter.findUnique({
+          where: {
+            id: Number(chapterId),
+          },
+        });
+
+        if (existingChapter) {
+          // Check if the user inputs are valid
+          const chapterUsers = await prisma.chapterUser.findMany({
```

Comment:
Chapter ID for chapter users is already included in the request session so this query is unnecessary. #60 includes code that performs this check so I recommend rebasing and re-using this logic.

Reply:
I think this query is still needed because we need to return the chapter information queried by chapterId later.

Reply:
you can make a query for the chapter, no need to fetch all chapter users